src/main/groovy/com/twcable/grabbit/resources/ContentResource.groovy

```diff
@@ -28,7 +28,7 @@ import javax.annotation.Nonnull
  * Queried from {@link com.twcable.grabbit.server.GrabbitContentPullServlet}.
  */
 @CompileStatic
-class ContentResource extends SyntheticResource {
+class ContentResource extends RootResource {
 
     public static final String CONTENT_RESOURCE_TYPE = "twcable:grabbit/content"
```

Comment:
You can now do 

`public static final String CONTENT_RESOURCE_TYPE = "${ROOT_RESOURCE_TYPE}/content"`